### ISSUE TITLE
License missing

### ISSUE THREAD

**Issue by ReneVolution** [pinglossy/smoothget#1]
Hi,

i would like to reuse your code for one of my clients but can't find anything about the license state of this project. Also, i was not able to send you an e-mail as it seems you didn't provide one which is why i'm opening this as issue. Would be great if you could contact me according on which license your code is published.

I also found a little bug which makes the build fail. I'll send you a pull request for that. And, i have actually trouble with capturing a live stream. Would be cool if you could guide me about what / where to change.

Best Regards,
René

**Comment by ile** [comment]
:+1: 

Do you have a license for this?

**Comment by Lazza** [comment]
Yeah, what about the license? By the way, the output looks a lot like the one of `ismdownloader`. Has there been any inspiration from it?

**Comment by jellelicht** [comment]
Any chance an update can given about the license for this code?

**Comment by Lazza** [comment]
I decompiled ismdownloader myself and noticed many portions of code which
were copied verbatim. I'd say it's unlikely this tool will ever be released
as free software.
